minos/common/model/declarative/aggregate/model.py

```diff
@@ -208,23 +208,29 @@ async def update(self: T, **kwargs) -> T:
         previous = await self.get_one(
             self.uuid, _broker=self._broker, _repository=self._repository, _snapshot=self._snapshot
         )
-        diff = AggregateDiff.from_difference(self, previous)
-        entry = await self._repository.update(diff)
+        aggregate_diff = AggregateDiff.from_difference(self, previous)
+        entry = await self._repository.update(aggregate_diff)
 
         self.uuid, self.version = entry.aggregate_uuid, entry.version
-        diff.uuid, diff.version = entry.aggregate_uuid, entry.version
+        aggregate_diff.uuid, aggregate_diff.version = entry.aggregate_uuid, entry.version
 
-        await self._send_update_events(diff)
+        await self._send_update_events(aggregate_diff)
 
         return self
 
-    async def _send_update_events(self, diff: AggregateDiff):
+    async def _send_update_events(self, aggregate_diff: AggregateDiff):
+        from ...dynamic import (
```

Comment:
I guess this import is here because of circular import, but I don't like to mix them with code. Have you tried to import it with an absolute route? If it's to complex I don't mind, just to know.

Reply:
Yes! That's the reason, but in this case I realised that there aren't any circular dependencies. In any case, I'm starting to think that some parts of `minos.model` are so "microservice"-agnostic and another ones are so "microservice"-related (`Entity`, `Aggregate`, `ValueObject`, `Event`, `Command` etc.), so in the future we might split this module into two ones to avoid these circular dependencies and keep the code clean.